sdk/documentintelligence/ai-document-intelligence-rest/src/outputModels.ts

```diff
@@ -69,33 +66,31 @@ export interface DocumentModelDetailsOutput {
   /** Document model description. */
   description?: string;
   /** Date and time (UTC) when the document model was created. */
-  createdDateTime: string;
+  readonly createdDateTime: string;
   /** Date and time (UTC) when the document model will expire. */
-  expirationDateTime?: string;
+  readonly expirationDateTime?: string;
   /** API version used to create this document model. */
-  apiVersion?: string;
+  readonly apiVersion?: string;
   /** List of key-value tag attributes associated with the document model. */
   tags?: Record<string, string>;
-  /**
-   * Custom document model build mode.
-   *
-   * Possible values: "template", "neural"
-   */
-  buildMode?: string;
+  /** Custom document model build mode. */
+  readonly buildMode?: DocumentBuildModeOutput;
```

Comment:
lost the possible values here.

Reply:
Good catch, @MaryGao @qiaozha I think we need to bring these back.